package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "com.team-zomsa.aplib-unity",
-    "version": "1.1.0",
+    "version": "2.0.0",
```

Comment:
Would make this the same as the Aplib.Net version that is within the package. (Right now seems to be v1.11.0-dev.6 ?)